### PR TITLE
Add 1.0.2-wasabi reproducible build

### PR DESCRIPTION
**Pull request by diega**

### CODE REVIEW

File: rskj/1.0.2-wasabi/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+```
+$ docker run --rm rskj-wasabi-1.0.2-reproducible sha256sum /code/rskj/rskj-core/build/libs/rskj-core-1.0.2-WASABI-all.jar
+2fb6dcc0ab18e3c43a0f5073b1e11d514bd7ea74cd430e1e0bfca06d3126242a /code/rskj/rskj-core/build/libs/rskj-core-1.0.2-WASABI-all.jar
+$ docker run --rm rskj-wasabi-1.0.2-reproducible sha256sum /code/rskj/rskj-core/build/libs/rskj-core-1.0.2-WASABI-sources.jar
+7da09e5be1541b1a57234f65c8a543bdb6b9e9ddbedb830387915a9f4a72c31d /code/rskj/rskj-core/build/libs/rskj-core-1.0.2-WASABI-sources.jar
+$ docker run --rm rskj-wasabi-1.0.2-reproducible sha256sum /code/rskj/rskj-core/build/libs/rskj-core-1.0.2-WASABI.jar
+007c27b02c8cd64d47affac7d18d793457aaf15d1b7ee773bd02e44d55676144 /code/rskj/rskj-core/build/libs/rskj-core-1.0.2-WASABI.jar
+$ docker run --rm rskj-wasabi-1.0.2-reproducible sha256sum /code/rskj/rskj-core/build/libs/rskj-core-1.0.2-WASABI.pom
+ea7cdf523a6512e8800634daac08dc6fac83986b155fc86424f5acfcce6cdefc /code/rskj/rskj-core/build/libs/rskj-core-1.0.2-WASABI.pom
+```
```
Comment (lucasvuotto) [inline rskj/1.0.2-wasabi/README.md]: Can't you change this to run the 4 hashes in only one container, instead of creating and destroying it everytime?

Comment (diega) [reply rskj/1.0.2-wasabi/README.md]: I honestly don't know. If you throw me a hint I can change it for sure

Comment (lucasvuotto) [reply rskj/1.0.2-wasabi/README.md]: How about:

```
$ docker run --rm rskj-wasabi-1.0.2-reproducible sha256sum \
    /code/rskj/rskj-core/build/libs/rskj-core-1.0.2-WASABI-all.jar \
    /code/rskj/rskj-core/build/libs/rskj-core-1.0.2-WASABI-sources.jar \
    /code/rskj/rskj-core/build/libs/rskj-core-1.0.2-WASABI.jar \
    /code/rskj/rskj-core/build/libs/rskj-core-1.0.2-WASABI.pom
2fb6dcc0ab18e3c43a0f5073b1e11d514bd7ea74cd430e1e0bfca06d3126242a /code/rskj/rskj-core/build/libs/rskj-core-1.0.2-WASABI-all.jar
7da09e5be1541b1a57234f65c8a543bdb6b9e9ddbedb830387915a9f4a72c31d /code/rskj/rskj-core/build/libs/rskj-core-1.0.2-WASABI-sources.jar
007c27b02c8cd64d47affac7d18d793457aaf15d1b7ee773bd02e44d55676144 /code/rskj/rskj-core/build/libs/rskj-core-1.0.2-WASABI.jar
ea7cdf523a6512e8800634daac08dc6fac83986b155fc86424f5acfcce6cdefc /code/rskj/rskj-core/build/libs/rskj-core-1.0.2-WASABI.pom
```

Comment (ida3m0n) [reply rskj/1.0.2-wasabi/README.md]: This will be changed on next release for sure. Moving forward

**Pull request merged**